AISKU/src/Initialization.ts

```diff
@@ -506,6 +508,11 @@ export class Initialization implements IApplicationInsights {
         }
     }
 
+    public getSender(): Sender {
+        let _self = this;
```

Comment:
It doesn't make sense (from a minification perspective) to have a local ```_self``` and assignment here... just use 
```return this._sender```